src/app/connect-wallet-alchemy/connect-wallet-alchemy.component.html

```diff
@@ -0,0 +1,83 @@
+<div class="h-screen bg-gradient-to-b from-green-100 to-brand-turquoise">
+  <div class="p-4">
+    <div class="flex items-center justify-between gap-6">
+      <img src="../assets/venture-miner-logo.png" class="w-32 bg-wite" />
+      <div class="flex flex-row items-center gap-4 text-sm">
```

Comment:
![image](https://github.com/Venture-Miner/w3fs/assets/111710515/20981d7d-8e57-4c38-94e6-935739c5db5a)

When I disconnect the wallet, the application keeps me "logged in". I'm not 100% positive but I imagine this is not the behaviour we want.